examples/web-chat/src/Room.tsx

```diff
@@ -77,39 +59,11 @@ export default function Room(props: Props) {
         title="Waku v2 chat app"
       />
       <ChatList messages={props.messages} />
-      <MessageInput
-        sendMessage={
-          waku
-            ? async (messageToSend) => {
-                return handleMessage(
-                  messageToSend,
-                  props.nick,
-                  props.commandHandler,
-                  async (msg) => {
-                    await waku.lightPush.push(ChatEncoder, msg);
-                  }
-                );
-              }
-            : undefined
-        }
-      />
+      <MessageInput hasPeers={!!lightPushPeers} sendMessage={onSend} />
```

Comment:
```suggestion
      <MessageInput hasLightPushPeers={!!lightPushPeers} sendMessage={onSend} />
```

Might be interested to specify that this variable only track light push peers